.github/workflows/Nuget-publishing.yml

```diff
@@ -0,0 +1,141 @@
+name: Nuget 📦 Distribution
+
+on:
+  workflow_run:
+    workflows:
+      - "Build UC2"
```

Comment:
I think we don't need to reply on this action because it's slow in most cases. You could just build binaries in this workflow though it might introduce some duplication.

Reply:
Wouldn't that take exactly as long as the workflow above?